### DISCUSSION TITLE
Sensitive info pushed to private repo

### DISCUSSION THREAD

**Discussion by robyn3choi** [community/community#21988] [New to GitHub]
<p>I recently pushed sensitive data to a private repo that was owned by my friend. He deleted the repo shortly afterward. We were the only collaborators on that repo, but I was wondering if there’s any chance at all that data could be detected by anyone else?</p>

**Comment by jfullstackdev** [comment, marked as answer]
<p>you may want to read this</p>
<aside class="onebox allowlistedgeneric" data-onebox-src="https://docs.github.com/en/site-policy/privacy-policies/github-privacy-statement">
  <header class="source">
      <img src="https://docs.github.com/assets/cb-803/images/site/favicon.svg" class="site-icon" width="16" height="16">

      <a href="https://docs.github.com/en/site-policy/privacy-policies/github-privacy-statement" target="_blank" rel="noopener">GitHub Docs</a>
  </header>

  <article class="onebox-body">
    <img src="https://user-images.githubusercontent.com/47092464/181068054-e35c4b16-909e-4bbf-9631-5936b70deaf7.png" class="thumbnail onebox-avatar" width="500" height="500" srcset="https://aws1.discourse-cdn.com/github/optimized/2X/3/337b7cc06184984e148c075df20ef009e0e6ecaa_2_500x500.png, https://aws1.discourse-cdn.com/github/optimized/2X/3/337b7cc06184984e148c075df20ef009e0e6ecaa_2_750x750.png 1.5x, https://aws1.discourse-cdn.com/github/optimized/2X/3/337b7cc06184984e148c075df20ef009e0e6ecaa_2_1000x1000.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/2X/3/337b7cc06184984e148c075df20ef009e0e6ecaa_2_10x10.png">

<h3><a href="https://docs.github.com/en/site-policy/privacy-policies/github-privacy-statement" target="_blank" rel="noopener">GitHub Privacy Statement - GitHub Docs</a></h3>

  <p>Get started, troubleshoot, and make the most of GitHub. Documentation for new users, developers, administrators, and all of GitHub's products.</p>


  </article>

  <div class="onebox-metadata">
    
    
  </div>

  <div style="clear: both"></div>
</aside>

**Comment by airtower-luna** [comment]
<p>Deleted repositories can be restored for a certain time, so while the data shouldn’t be accessible any more it’s still stored:</p><aside class="onebox allowlistedgeneric" data-onebox-src="https://docs.github.com/en/repositories/creating-and-managing-repositories/restoring-a-deleted-repository">
  <header class="source">
      <img src="https://docs.github.com/assets/cb-803/images/site/favicon.svg" class="site-icon" width="16" height="16">

      <a href="https://docs.github.com/en/repositories/creating-and-managing-repositories/restoring-a-deleted-repository" target="_blank" rel="noopener">GitHub Docs</a>
  </header>

  <article class="onebox-body">
    <img src="https://user-images.githubusercontent.com/5814919/181068056-959ae934-7ac6-47e1-952a-87d976b00b59.png" class="thumbnail onebox-avatar" width="500" height="500" srcset="https://aws1.discourse-cdn.com/github/optimized/2X/3/337b7cc06184984e148c075df20ef009e0e6ecaa_2_500x500.png, https://aws1.discourse-cdn.com/github/optimized/2X/3/337b7cc06184984e148c075df20ef009e0e6ecaa_2_750x750.png 1.5x, https://aws1.discourse-cdn.com/github/optimized/2X/3/337b7cc06184984e148c075df20ef009e0e6ecaa_2_1000x1000.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/2X/3/337b7cc06184984e148c075df20ef009e0e6ecaa_2_10x10.png">

<h3><a href="https://docs.github.com/en/repositories/creating-and-managing-repositories/restoring-a-deleted-repository" target="_blank" rel="noopener">Restoring a deleted repository - GitHub Docs</a></h3>

  <p>You can restore some deleted repositories to recover their contents.</p>


  </article>

  <div class="onebox-metadata">
    
    
  </div>

  <div style="clear: both"></div>
</aside>

<p>Your friend could contact support and ask them to delete the data earlier.</p><aside class="onebox allowlistedgeneric" data-onebox-src="https://support.github.com/request">
  <header class="source">
      <img src="https://user-images.githubusercontent.com/5814919/181068058-87fd5e49-9e5d-4be5-9bde-2e1f6425be07.png" class="site-icon" width="32" height="32">

      <a href="https://support.github.com/request" target="_blank" rel="noopener">support.github.com</a>
  </header>

  <article class="onebox-body">
    

<h3><a href="https://support.github.com/request" target="_blank" rel="noopener">GitHub Support</a></h3>

  <p>GitHub Support is here to help. Learn about GitHub products, browse our helpful resources, and contact support with your questions.</p>


  </article>

  <div class="onebox-metadata">
    
    
  </div>

  <div style="clear: both"></div>
</aside>
